BaseClasses.py

```diff
@@ -767,8 +767,25 @@ def item_count(self, item: str, player: int) -> int:
         Utils.deprecate("Use count instead.")
         return self.count(item, player)
 
+    def has_list(self, items: Iterable[str], player: int, count: int = 1) -> bool:
```

Comment:
I'm wondering if this should remove the default of `1` since, as far as I can tell, if you're actually using `1` you should just be using `has_any` instead